apps/cookbook/src/app/app-routing.module.ts

```diff
@@ -7,6 +7,7 @@ import { routes } from './app.routes';
   imports: [
     RouterModule.forRoot(routes, {
       scrollPositionRestoration: 'enabled',
+      relativeLinkResolution: 'legacy',
```

Comment:
Do we know why it's set to `legacy` and not `corrected` (which should be the default)?:

> The default in v11 is `corrected`.

https://angular.io/api/router/ExtraOptions#relativeLinkResolution